src/config/chains.ts

```diff
@@ -1,3 +1,8 @@
+export enum NetworkNameEvm {
```

Comment:
nit: NetworkName is not just for EVM (since likely it'll expand to handle all networks?)

Reply:
The SDK itself is only for EVM. Do we have any plan to extend this SDK for non-EVM networks?

Reply:
Because it is for EVM we don't need to distinguish that it's EVM